### ISSUE TITLE
It would be great to see us extend these H-tests to all years on record.

### ISSUE THREAD

**Issue by judecalvillo** [sfbrigade/CA_DOJ_OpenJustice#7]

**Comment by judecalvillo** [comment]
This is for Prompt #2, btw. It looks like GitHub projects doesn't supply a note/issue category label to issues when they originate from the 'Projects' tab.

**Comment by billiescodes** [comment]
Hi Jude! That's a great step to pick up this project again. Let me start work on that. See you on Wednesday

**Comment by judecalvillo** [comment]
Thanks, @billiescodes. Yeah, @catherinemzhang has already done a great job starting the H-tests (in Python). That said, all we gotta' do is extend this to all years, maybe add some color with the other stuff you've done, and, of course, add whatever full-span, within-span, and between sub-segment tests/plots you think the public would appreciate. We should have more cool stuff to do for/with the DoJ in the coming weeks, as we recently had a great call with OpenJustice's new head of research. See you on Wednesday!

**Comment by VincentLa** [comment]
Hi all! I'm new to both the group and the project. I'm going to see if I can get more context and pick up this issue?

**Comment by judecalvillo** [comment]
Hey Vincent, 

Great to hear of your interest. I'm sorry I can't be there, but I've been overwhelmed by work and have another deadline tomorrow morning. 

Regarding this specific issue, it's currently in a holding pattern, because the new/full dataset is now 4.5 Gb in size. That's too large to work with from our current Notebook host, but we'll soon be deploying our own Notebook hosting. Stay tuned and, by all means, feel free to read through past Notebooks to get caught up on the relevant context. 

Looking forward to working with you. 

- Jude 

---- On Wed, 22 Feb 2017 20:35:26 -0800 notifications@github.com wrote ----

Hi all! I'm new to both the group and the project. I'm going to see if I can get more context and pick up this issue?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.


Confidentiality Notice: This email message (including any attachments) may contain confidential, privileged or non-public information and is for the sole use of the designated recipient(s). If you received this communication in error, please notify the sender immediately by replying to this message and then delete the original message and any attachments from your system. Use, dissemination, distribution, or reproduction of this message by unintended recipients is not authorized and may be unlawful.

**Comment by VincentLa** [comment]
Got it. There's also some analysis to be done on adults as well right? Maybe I can get started on that if I have free time.

**Comment by judecalvillo** [comment]
Yeah, their initial focus was juveniles. However, I believe their impending _transactional_ (post-arrest interactions with the justice system) inquiries won't be focused on juveniles. Plus, it's likely that, if we offer to look into adult arrest stats, they'd be interested. To be discussed.




---- On Thu, 23 Feb 2017 09:58:27 -0800 Vincent La &lt;notifications@github.com&gt; wrote ----




Got it. There's also some analysis to be done on adults as well right? Maybe I can get started on that if I have free time.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.








Confidentiality Notice: This email message (including any attachments) may contain confidential, privileged or non-public information and is for the sole use of the designated recipient(s). If you received this communication in error, please notify the sender immediately by replying to this message and then delete the original message and any attachments from your system. Use, dissemination, distribution, or reproduction of this message by unintended recipients is not authorized and may be unlawful.
